### DISCUSSION TITLE
Link to Sonarr/Radarr page for media item

### DISCUSSION THREAD

**Discussion by Weavus** [sct/overseerr#549] [Feature Requests]
I don't see any way currently to be able to go directly to the Radarr or Sonarr page for the media item listed. Could you add a link to the entry in Radarr/Sonarr on the item detail page?

**Comment by sct** [comment]
This is planned to come at some point after Radarr/Sonarr sync is added.

**Comment by edsilberberg** [comment]
I saw the 'Play on Plex' button  and not long after looked for an 'Open in Radarr' button.

Seconding this request and glad to see on the roadmap already.

**Reply by TheCatLady** [reply]
This already exists in the settings slideover (click the gear button).

**Reply by edsilberberg** [reply]
Aha, found it, I first checked global settings but then found the gear.
I hereby amend my request, add 'Open in Radarr' to the dropdown as a choice before 'Play trailer'

**Reply by TheCatLady** [reply]
I disagree.  The purpose of the button is to play media, and the *arrs do not fit there.

Also, adding the *arr links to the dropdown does not reduce the number of clicks.  Just click the cog button instead of the dropdown arrow?
